op-batcher/batcher/driver.go

```diff
@@ -414,3 +412,19 @@ func (l *BatchSubmitter) l1Tip(ctx context.Context) (eth.L1BlockRef, error) {
 	}
 	return eth.InfoToL1BlockRef(eth.HeaderBlockInfo(head)), nil
 }
+
+func logFields(xs ...any) (fs []any) {
+	for _, x := range xs {
+		switch v := x.(type) {
+		case txData:
+			fs = append(fs, "fid", v.ID(), "data_len", v.Len())
```

Comment:
```suggestion
			fs = append(fs, "frame_id", v.ID(), "data_len", v.Len())
```
I'm assuming `fid` is frame ID here

Reply:
I meant to use `fid` for frame id, yeah. It's not used anywhere else in logging, so I just established `fid` for frame id. But I like the more verbose field name `frame_id`, will change. In a follow-up, we can also streamline logging of channels as `channel_id`.